contracts/blade/BridgeStorage.sol

```diff
@@ -54,7 +54,15 @@ contract BridgeStorage is ValidatorSetStorage {
 
         bytes memory hash = abi.encode(
             keccak256(
-                abi.encode(batch.rootHash, batch.startId, batch.endId, batch.sourceChainId, batch.destinationChainId)
+                abi.encode(
+                    batch.threshold,
```

Comment:
Put these fields last in the batch. Whenever we add some new fields, always put them last, not just in signing, but in the struct itself.